p2p/src/protocol/cbfmgr.rs

```diff
@@ -249,7 +249,10 @@ pub trait Events {
 pub enum GetFiltersError {
     /// The specified range is invalid, eg. it is out of bounds.
     #[error("the specified range is invalid, current tip is {tip}")]
-    InvalidRange { tip: Height },
+    InvalidRange {
+        /// Filter header tip
```

Comment:
meaningless? The code here is clear, right?

Reply:
There's a `missing_docs deny` [rule](https://github.com/cloudhead/nakamoto/blob/9f5ea461f8fbe37d36d2a6dc336af61abb3dd8ba/p2p/src/lib.rs#L25)

Reply:
Oh right, forgot about it!